code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -802,11 +802,46 @@
 						continue
 					M.reagents.add_reagent("jenkem", 1)
 
+		if("hem")
+			message = "<b>[src]</b> hems."
```

Comment:
Lifeweb emotes =V

I never understood this one, to be honest; it's a term I've quite literally never heard outside of lifeweb.

People go "ahem", or "mhmmm", but they don't just go "hemmmmmm".